### DISCUSSION TITLE
Prodigy killed during training

### DISCUSSION THREAD

**Discussion by alvaromarlo** [explosion/spaCy#12004] [Help: Other Questions]
Hi,

First of all I don't know if this is a bug related to spaCy, but I would like to know if you can give me a hand.

When I run the training, it stops and returns this message:

`{route-to-my-project-env}/bin/prodigy: line 2: 22878 Killed: 9   python -m prodigy "$@"`

Any idea what it could be due to?

Thanks,
Álvaro

**Comment by polm** [comment]
If your process is being killed and you didn't do it yourself, I'd guess it's due to running out of memory. Do you have enough data that that sounds plausible?

You should be able to check if a process was killed due to running out of memory with a command like this:

```
cat /var/log/messages | grep Kill
```
